tests/test_feature_extractor_static.py

```diff
@@ -9,13 +10,33 @@ def setUp(self):
         path_struct = "/Users/ehuber/Documents/ambiguous_alpaca/ambigous-alpaca/german-structgram-mincount-30-ctx-10-dims-300.fifu"
         self.extractor_skipgram = StaticEmbeddingExtractor(path_skipgram)
         self.extractor_structgram = StaticEmbeddingExtractor(path_struct)
+        self.emb_skip = self.extractor_skipgram.get_embedding("Pferd")
+        self.emb_struct = self.extractor_structgram.get_embedding("Pferd")
+        array_words = ["Schaaf", "Pferd", "Hund", "Katze"]
+        self.array_embeddings = self.extractor_skipgram.get_array_embeddings(array_words)
 
     def test_dimension(self):
+        """
+        test dimension of both types of embedding (skipgram and structured skipgrams)
+        """
         expected = 300
-        emb = self.extractor_skipgram.get_embedding("Pferd")
-        #...
-    def test_distance(self):
-        pass
+
+        np.testing.assert_equal(expected, self.emb_skip.shape[0])
+        np.testing.assert_equal(expected, self.emb_struct.shape[0])
+
+    def test_embeddings_diferent(self):
+        """
+        tests that two different embeddings (structured vs. normal skipgram) correspond to the the same word
+        """
+        np.testing.assert_raises(AssertionError, np.testing.assert_array_equal, self.emb_skip, self.emb_struct)
```

Comment:
it is nicer not to test that an assertion error is raised but that some equal returns False. So I would do 
np.testing.assert_equal( np.array_equal(self.embed_skip , self.emb_struct), False)